store.go

```diff
@@ -3328,6 +3361,28 @@ func (s *store) ContainerDirectory(id string) (string, error) {
 	return res, err
 }
 
+func (s *store) ImageRunDirectory(id string) (string, error) {
+	foundImage := false
+	if res, done, err := readAllImageStores(s, func(store roImageStore) (string, bool, error) {
+		if store.Exists(id) {
+			foundImage = true
+		}
+
+		middleDir := s.graphDriverName + "-images"
+		rcpath := filepath.Join(s.RunRoot(), middleDir, id, "userdata")
```

Comment:
(Yes the API documents that these directories are not explicitly deleted. A small comment here might help, e.g. for refactoring efforts which don’t naturally end up reading that API documentation.)